tests/common.py

```diff
@@ -0,0 +1,121 @@
+import logging
+import json
+import os
+
+import pyarrow as pa
+from collections.abc import Iterable
+
+# create and configure main logger
+LOGGER = logging.getLogger(__name__)
+LOGGER.setLevel(logging.INFO)
+
+# create console handler with a higher log level
+LOG_HANDLER = logging.StreamHandler()
+LOG_HANDLER.setLevel(logging.DEBUG)
+
+# create formatter and add it to the handler
+FORMATTER = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+LOG_HANDLER.setFormatter(FORMATTER)
+
+# add the handler to the logger
+logging.getLogger('').addHandler(LOG_HANDLER)
+
+REALPATH_DIRECTORY = os.path.realpath(os.path.join(os.getcwd(), os.path.dirname(__file__)))
+
+
+def get_full_path(file_names: Iterable[str]) -> list[str]:
+    data_dir = os.path.join(REALPATH_DIRECTORY, 'data/tpch_parquet')
+    full_paths_list = (
+        [os.path.join(data_dir, dataset) for dataset in file_names]
+    )
+
```

Comment:
we could get rid of this spacing?

Reply:
I would recommend setting up `black` and `isort` and we can skip over any remaining questions around formatting and linting.

Reply:
Thanks for the suggestion!  Just set those 2 up on my IDE

Reply:
probably we can add a style checker somwhere down the road to check them for each PR.